djchoices/tests/test_choices.py

```diff
@@ -65,6 +67,7 @@ def test_class_values(self):
         self.assertEqual(SubClass1.values[SubClass1.Item_1], "Item 1")
         self.assertEqual(SubClass1.values[SubClass1.Item_4], "Item 4")
         self.assertEqual(SubClass1.values[SubClass1.Item_5], "Item 5")
+        self.assertIsInstance(SubClass1.values, OrderedDict)
```

Comment:
Cool, much appreciated!

As for the style of the tests - correctness is most important, then readability and then all the rest :-)